mso/resource_mso_schema_template_external_epg_subnet.go

```diff
@@ -50,11 +50,13 @@ func resourceMSOTemplateExtenalepgSubnet() *schema.Resource {
 				Type:         schema.TypeString,
 				Required:     true,
 				ValidateFunc: validation.StringLenBetween(1, 1000),
+				ForceNew:     true,
 			},
 			"name": &schema.Schema{
 				Type:         schema.TypeString,
 				Optional:     true,
 				ValidateFunc: validation.StringLenBetween(1, 1000),
+				Default:      nil,
```

Comment:
Do we need this default or should we have a else condition later?

Reply:
added a condition in mso-go-client to check whether Name attr exist